angularSTEP/src/app/navbar/navbar.component.html

```diff
@@ -0,0 +1,18 @@
+<mat-toolbar color="primary" class="mat-elevation-z8">
+    <span>Recipe Stuff</span>
+
+    <a mat-button>Home</a>
+    <a mat-button>Discover</a>
+    <a mat-button>Recipes</a>
+
+    <div class="spacer">
+        <mat-form-field>
+            <mat-label><mat-icon>search</mat-icon></mat-label>
+            <input matInput>
+        </mat-form-field>
+    </div>
+
+    
+    <a mat-raised-button color="accent">Sign Up</a>
+    <a mat-raised-button color="red">Log In</a>
```

Comment:
I would say that you should make these mat-buttons because it would look a lot cleaner and then you wouldn't need to add functionality in the scss to make hovering and clicking work. However, the menu dropdown component that I made would be able to be added to this and would remove the need for these buttons. Therefore, I don't think it is vital to change right now.

Reply:
The styling is mainly for the underline effect, and hover change effects. I am not sure how your menu drop down would be implemented. It might be best for the change to come from your end when you are putting in the drop down menu. just a suggestion.